### PR TITLE
translate: [api/interfaces/] 4 following files

### PR DESCRIPTION
**Pull request by icarusgk**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Feature :sparkles:
- [ ] Code style update (formatting, renaming) :art:
- [ ] Refactoring (no functional changes, no api changes) :recycle:
- [x] Documentation content changes :memo:
- [ ] Other :zap:

## What is the new behavior?

_Translate the following files:_
- `pinia.DefineStoreOptionsInPlugin.md`
- `pinia.MapStoresCustomization.md`
- `pinia.Pinia.md`
- `pinia.PiniaCustomProperties`

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: api/interfaces/pinia.DefineStoreOptionsInPlugin.md [hunk 1]
```diff
@@ -6,84 +6,81 @@ sidebarDepth: 3
 
 [API Documentation](../index.md) / [pinia](../modules/pinia.md) / DefineStoreOptionsInPlugin
```
Comment (edgonzalez24) [inline api/interfaces/pinia.DefineStoreOptionsInPlugin.md:7]: Missing translation to "Documentación de la API"

File: api/interfaces/pinia.DefineStoreOptionsInPlugin.md [hunk 2]
```diff
@@ -6,84 +6,81 @@ sidebarDepth: 3
...
 ___
 
-### getters
+### getters {#getters}
 
-• `Optional` **getters**: `G` & `ThisType`<`UnwrapRef`<`S`\> & [`_StoreWithGetters`](../modules/pinia.md#_storewithgetters)<`G`\> & [`PiniaCustomProperties`](pinia.PiniaCustomProperties.md)<`string`, [`StateTree`](../modules/pinia.md#statetree), [`_GettersTree`](../modules/pinia.md#_getterstree)<[`StateTree`](../modules/pinia.md#statetree)\>, [`_ActionsTree`](../modules/pinia.md#_actionstree)\>\> & [`_GettersTree`](../modules/pinia.md#_getterstree)<`S`\>
+• `Opcional` **getters**: `G` & `ThisType`<`UnwrapRef`<`S`\> & [`_StoreWithGetters`](../modules/pinia.md#_storewithgetters)<`G`\> & [`PiniaCustomProperties`](pinia.PiniaCustomProperties.md)<`string`, [`StateTree`](../modules/pinia.md#statetree), [`_GettersTree`](../modules/pinia.md#_getterstree)<[`StateTree`](../modules/pinia.md#statetree)\>, [`_ActionsTree`](../modules/pinia.md#_actionstree)\>\> & [`_GettersTree`](../modules/pinia.md#_getterstree)<`S`\>
 
-Optional object of getters.
+Objecto opcional de getters.
```
Comment (edgonzalez24) [inline api/interfaces/pinia.DefineStoreOptionsInPlugin.md:46]: "Objeto" instance of "Objecto"

File: api/interfaces/pinia.MapStoresCustomization.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ sidebarDepth: 3
 
 [API Documentation](../index.md) / [pinia](../modules/pinia.md) / MapStoresCustomization
```
Comment (edgonzalez24) [inline api/interfaces/pinia.MapStoresCustomization.md:7]: Missing translation to "Documentación de la API"

File: api/interfaces/pinia.DefineStoreOptionsInPlugin.md (hunk 1 above)
Comment (icarusgk) [reply api/interfaces/pinia.DefineStoreOptionsInPlugin.md:7]: Fixed 🚀 Gracias por notarlo

**Review by edgonzalez24 — APPROVED** [review]
LGTM!

**Pull request merged**